workflow/rules/pseudoassembly.smk

```diff
@@ -24,7 +24,7 @@ rule compare_assemblies:
         assembly="results/{date}/contigs/polished/{sample}.fasta",
         pseudoassembly="results/{date}/contigs/pseudoassembled/{sample}.fasta",
     output:
-        "results/{date}/aligned/assemblies/{sample}.bam",
+        temp("results/{date}/aligned/assemblies/{sample}.bam"),
```

Comment:
same here

Reply:
I think here we can delete the bam file, as the rule `aggregate_assembly_comparisons` parses it and creates a more human readable output from it.